source/common/access_log/access_log_manager_impl.h

```diff
@@ -83,6 +83,9 @@ class AccessLogFileImpl : public AccessLogFile {
   void open();
   void createFlushStructures();
 
+  // return default flags set which used by open
+  Filesystem::FlagSet defaultFlags();
```

Comment:
Declare this method static; it does not appear to require this.

Reply:
Thanks and will do.